docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+
+services:
+  app:
+    build: .
+    restart: always
+    image: "iit-backend"
+    ports:
+      - "5000:5000"
+    env_file:
+      - .env
+    environment:
+      - FLASK_ENV=prod
```

Comment:
What if this is run locally?

Reply:
This env var is only set because it triggers us to use `gunicorn` to start up the app instead of `flask run`. I was having trouble on the networking side working with flask run (for some reason even though I exposed port 5000 the app wasn't accessible by my host machine).

I will put in a comment describing this